### DISCUSSION TITLE
Limits of answers

### DISCUSSION THREAD

**Discussion by szurmandev** [pj8912/php-bard-api#9] [General]
Are there any limits to BARD's responses? I query from every 60 seconds for a response, I received 126 responses and then returned blank, I tried so another google account (I changed the PSID API KEY) and it responded 7 and then returned blank result too. Am I doing something wrong or is google limiting this somehow?

**Comment by pj8912** [comment]
Well it's not an official API, it does take long sometimes. increase delay time for response if you want to.
